com.unity.netcode.gameobjects/Tests/Editor/NetworkManagerMessageHandlerTests.cs

```diff
@@ -1,6 +1,8 @@
 using System;
 using NUnit.Framework;
+#if UNITY_EDITOR
```

Comment:
Editor tests aren't run as part of the mobile CI - is this actually still needed or is it left-over from when we were trying to get the editor tests running?

Reply:
Same question for several other files in this changeset which don't seem like they should be affected/included in here:

`com.unity.netcode.gameobjects/Tests/Editor/NetworkObjectTests.cs`
`com.unity.netcode.gameobjects/Tests/Editor/Timing/NetworkTimeTests.cs`
`com.unity.netcode.gameobjects/Tests/Editor/com.unity.netcode.editortests.asmdef` (this one is also only a whitespace change)